flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
why?

Reply:
Not sure actually, couldn't test it myself. The project should in theory work on aarch64, will remove the flathub.json file. Perhaps i can always limit it later when users are reporting it not to be working?